.vscode/launch.json

```diff
@@ -66,6 +66,25 @@
 			"console": "integratedTerminal",
 			"internalConsoleOptions": "neverOpen"
 		},
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "Debug Clean Build in Selected Project (Heft)",
```

Comment:
Is this useful as a separate entry, or should we just add `--clean` to `"Debug Build in Selected Project (Heft)"`